bot/cogs/snekbox.py

```diff
@@ -196,6 +219,15 @@ def get_status_emoji(results: dict) -> str:
             await ctx.invoke(self.bot.get_command("help"), "eval")
             return
 
+        if action == "-save":
```

Comment:
This entire thing appears to be a poor-mans subcommand handler. There's no reason to parse distinct flags contrary to the entire commands framework design when we have the ability to add subcommands through usage of a command group object.